packages/desktop-client/src/components/ManageRules.js

```diff
@@ -179,18 +178,9 @@ export function Value({
         {numHidden > 0 && (
           <Text style={{ color: colors.p4 }}>
             &nbsp;&nbsp;
-            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-            <a
-              href="#"
-              onClick={onExpand}
-              {...css({
-                color: colors.p4,
-                textDecoration: 'none',
-                ':hover': { textDecoration: 'underline' },
-              })}
-            >
+            <Link onClick={onExpand} style={{ color: colors.p4 }}>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏(probably not in scope for your PR): I find the name of `Link` component ambiguous. It implies that it's internally using `<a>`, but instead it's using a button. Would be nice to rename it to `ButtonLink` so it's more apparent what this actually does. Especially because these "links" don't have a target, but only on-click handlers.

Reply:
Ah right, I had meant to rename it. Can do!